apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1363,920 +1370,661 @@ describe('Ticket', () => {
         })
     })
 
-    describe( 'Notification', () => {
```

Comment:
Why do you want to delete tests?

Reply:
These are ticketComment tests, I've moved them to corresponding file.
There also were two tests for case when executor is the same with assignee in ticket, we discussed this logic with Mikhail Rumanovsky, and decided to remove it, so those tests are not needed any more.